lib/MiqVm/MiqRhevmVm.rb

```diff
@@ -51,12 +52,17 @@ def getCfg(_snap = nil)
       storage_obj    = storage_domains_by_id[storage_id]
 
       file_path = file_path_for_storage_type(storage_obj, disk)
+      file_path_s = file_path.to_s
 
       tag = "scsi0:#{idx}"
       cfg_hash["#{tag}.present"]    = "true"
       cfg_hash["#{tag}.devicetype"] = "disk"
-      cfg_hash["#{tag}.filename"]   = file_path.to_s
+      cfg_hash["#{tag}.filename"]   = file_path_s
       cfg_hash["#{tag}.format"]     = disk.format
+
+      if storage_obj.storage.type == "ISCSI" && !lv_active?(file_path_s)
+        AwesomeSpawn.run!("sudo lvchange", :params => [:activate, "y", file_path_s])
```

Comment:
@nasark do we need to de-activate the lv once SSA is complete?

Reply:
I don't think that's the expectation